.changeset/breezy-tips-walk.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/core': patch
+---
+
+Improved project correlation in server telemetry without changing existing project identifiers.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)